### DISCUSSION TITLE
Watch Environment Variables on windows

### DISCUSSION THREAD

**Discussion by mathysd** [lerna/lerna#3663] [Q&A] [answered]
Are the environment variables with the watch command supported on windows?

This commands works as expected on linux 

npx -c 'lerna watch -- echo \$LERNA_PACKAGE_NAME \$LERNA_FILE_CHANGES'

when making a change I get:

header packages/header/src/Header.tsx

When running the same command on windows and making a change I get:

\$LERNA_PACKAGE_NAME \$LERNA_FILE_CHANGES

**Comment by ghiscoding** [comment, marked as answer]
It might be missing in the docs but the `$` variables are Unix based only and not cross-platform, so a `$var` can only be read with `%var%` on Windows, or in your case that would be `%LERNA_FILE_CHANGES%`, if you want a cross-platform approach you can use `cross-env` lib

```sh
lerna watch -- cross-env-shell echo $LERNA_FILE_CHANGES
```
